### ISSUE TITLE
Addding The script support

### ISSUE THREAD

**Issue by modz2014** [activescott/lessmsi#181]
any chance of adding support to extract the script file that is used to create the installer example for Wix

**Comment by activescott** [comment]
Are you asking for a feature to provide the WIX script that was used to create a .MSI file?On Sep 1, 2023, at 19:58, modz2014 ***@***.***> wrote:﻿
any chance of adding support to extract the script file that is used to create the installer example for Wix

**Comment by modz2014** [comment]
yes addign the feature for it would be nice

**Comment by activescott** [comment]
As I recall the wix project published a "dark" utility that would allow getting the wix files. In theory that could be integrated into Lessmsi if someone wanted. If someone wants to take on the effort track down the latest wix project files and give me a heads up on your plan and I'll support you.

**Comment by modz2014** [comment]
@activescott yes im also looking it creating this into a 7zip plugin if your interested
